src/social_network_link_prediction/other_methods/neighbor_set_information.py

```diff
@@ -69,16 +68,14 @@ def likelihood(z, G):
 def MI(G : nx.Graph, lalla: int):
     I_Oxy = 0
     s_xy = []
+    res = np.zeros((G.number_of_nodes(), G.number_of_nodes()))
```

Comment:
La matrice risultante sarà piena o può anche essere sparsa ??

Nel caso sarebbe da utilizzare direttamente una matrice sparsa.

Reply:
Risolto in 1b04464